### ISSUE TITLE
Keep multiple movie versions

### ISSUE THREAD

**Issue by m4f1j0z0** [Radarr/Radarr#3832]
TL;DR: I would like to keep several version of a movie, not just the last downloaded one (e.g. 4K BluRay and a WEB-DL). 

I'm running an Emby server on a NAS, too weak to transcode. Several people using my server have slow Internet connections and cannot stream anything in 4K or higher bitrate 1080p.

I would like to be able to configure Radarr to keep several versions of a Movie, for instance a 4K BluRay and a WEB-DL version, that way people can choose which version they want to stream, no transcoding necessary. 

At the moment, when Radarr hits cutoff, replaces the version of inferior quality and downloads a 4K version of a movie, I have to manually copy an old WEB-DL release into the folder.

**Comment by Qstick** [comment]
Dupe of #1910
